docs/user-guide/ci-setup.md

```diff
@@ -67,6 +89,52 @@ run previously failing tests, additional label must be added to the PR (e.g.
 label `1`). In this case the test labels already added to the PR will prevent
 those tests from being re-executed.
 
+### Connecting to the Environment
+
+CI operators can use an Azure VPN to [connect to the local CI using P2S](https://learn.microsoft.com/azure/vpn-gateway/vpn-gateway-howto-point-to-site-resource-manager-portal#connect), certificates and secrets are in the Keyvault
+
+### Manual steps to recreate the environment
+
+In addition of running the Bicep scripts, the following action needs to be manually executed:
+
+1. Create a service principal with permissions as described [above](#pipeline-authentication-setup)
+1. Create a new Github Environment (Or update the existing one) to reflect the client id of the new service principal
```

Comment:
Does that mean updating AZURE_CLIENT_ID, AZURE_TENANT_ID and AZURE_SUBSCRIPTION_ID?

Reply:
Zes, I will be more specific

Reply:
Let me know if this is better now

Reply:
Yes, perfect.